app/lib/actions/pulp/consumer/update.rb

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2014 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+module Actions
+  module Pulp
+    module Consumer
+      class Update < Actions::EntryAction
```

Comment:
I think the same issue here, this needs to inherit from Pulp::Abstract too.  This seemed to work for me:

```
module Actions
  module Pulp
    module Consumer
      class Update < Pulp::Abstract
        input_format do
          param :uuid, String
          param :display_name, String
        end

        def plan(system)
          plan_self(:uuid => system.uuid, :display_name => system.name)
        end

        def run
          output[:response] = pulp_extensions.consumer.update(input[:uuid], :display_name => input[:name])
        end
      end
    end
  end
end
```